src/main/java/ua/danit/final_project/controllers/VacancyController.java

```diff
@@ -25,13 +26,16 @@ public VacancyController(VacancyService vacancyService) {
   }
 
   @GetMapping
-  public List<Vacancy> getVacancies() {
-    return vacancyService.getOpenVacancies();
+  public List<VacancyDto> getVacanciesDto() {
+    return vacancyService.getOpenVacancies()
+            .stream()
+            .map(vacancy -> new VacancyDto(vacancy))
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) ([squid:S1612](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1612))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=105&issues=AWXJ1tR8lgbYnohY1fRK&open=AWXJ1tR8lgbYnohY1fRK)

[ISSUE_KEY]: AWXJ1tR8lgbYnohY1fRK